composer.json

```diff
@@ -19,7 +19,8 @@
     "require": {
         "php": ">=5.3.0",
         "psr/log": "^1.0",
-        "symfony/var-dumper": "^2.6|^3.0"
+        "symfony/var-dumper": "^2.6|^3.0",
+        "symfony/debug": "2.8.*|3.0.*"
```

Comment:
I tested it using a laravel install, so I imported that version of symfony/debug because it was the one defined in laravel's composer.json and I knew it worked. This is why I forgot to add to composer.json initially - it was already a requirement of laravel so I didn't need to add it again.